crates/holochain_persistence_api/src/eav/query.rs

```diff
@@ -87,21 +86,16 @@ impl<'a, A: Attribute> EaviQuery<'a, A> {
                                     &self.attribute,
                                     &fold_query.value,
                                 ) {
-                                    //check if tombstone condition is met
+                                    //check if tombstone condition is met at
+                                    //if a tombstone is not found it should default to false
                                     if self
                                         .tombstone()
                                         .as_ref()
                                         .map(|s| s.check(eavi_fold.attribute()))
-                                        .unwrap_or(true)
+                                        .unwrap_or_default()
```

Comment:
Your change here is logically the same but simpler, right?

Reply:
The change here is more of a bug fix, if no tombstone is found the eavi_fold value should continue on until the latest is obtained

Reply:
Ah I get it. The `.map(|_| true)` function will "override" `s.check(eavi_fold.attribute())`if `tombstone()` is `Some(_)`.